.circleci/config.yml

```diff
@@ -5,6 +5,7 @@ jobs:
       - image: google/cloud-sdk
     environment:
       TEST_CLUSTER_ZONE: europe-west3-b
+      ACCEPT_EULA: "yes"
```

Comment:
Should there be " around yes?

Reply:
Getting syntax error when removing. Added " around europe-west3-b for consistency